doc/nrf/libraries/networking/nrf_cloud_alerts.rst

```diff
@@ -0,0 +1,108 @@
+.. _lib_nrf_cloud_alerts:
+
+nRF Cloud Alerts
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This library is an enhancement to the :ref:`lib_nrf_cloud` library.
+The nRF Cloud alerts library enables applications to generate and transmit messages that comply with the alerts feature of nRF Cloud.
+Alerts messages are designed to be compact JSON documents that indicate the occurrence of a critical event, such as a low battery or unexpected reset.
+
+Overview
+********
+
+This library provides an API for either REST-based or MQTT-based applications to send alerts to nRF Cloud.
+For MQTT-based applications, users can enable or disable alerts remotely using the nRF Cloud portal or REST API.
+
+Each alert contains the following elements, as defined in the structure :c:struct:`nrf_cloud_alert_info`:
+
+* :c:member:`type` - a with a value of the :c:enum:`nrf_cloud_alert_type` enum
+* :c:member:`value` - use 0 if not otherwise needed
+* :c:member:`description` - use NULL to suppress transmission of this field
+* :c:member:`sequence` - Sequence number used when no timestamp is available
+* :c:member:`ts_ms` - Timestamp if accurate time is known
+
+The range of the alert type enumeration is split into two parts:
+
+* Values less than :c:enumerator:`ALERT_TYPE_CUSTOM` are predefined
+* Values greater or equal to that are for application-specific use
+
+Use the alert value to indicate more information about an alert.
+For example, with the :c:enumerator:`ALERT_TYPE_TEMPERATURE`, it could represent the temperature in degrees Celcius.
+
+Use the description string to indicate more detail about a specific alert, if required.
+
+The nRF Cloud alerts library sets the sequence number and timestamp automatically.
+The nRF Cloud alerts service uses these values to present the alerts in the actual sequence they were generated on the device.
+
+Supported features
+==================
+
+If the :ref:`lib_date_time` library is enabled, and the current date and time are known, the timestamp field of the alert will be automatically set accordingly.
+If the :ref:`lib_date_time` library is not enabled, the sequence number will be set to a monotonically-increasing value.
+
+Requirements
+************
+
+The device must be connected to nRF Cloud prior to calling the :c:func:`nrf_cloud_alert_send` function.
+The :c:func:`nrf_cloud_rest_alert_send` function initiates the connection as needed.
+
+Configuration
+*************
+
+Configure the following options to enable or disable the use of this library:
```

Comment:
I changed it to this, since the first option listed is for enabling/disabling, but the second is to select the transport method:
`Configure the following options to enable or disable the library and to select the data transport method:`